feathr_project/setup.py

```diff
@@ -38,7 +38,7 @@
         # fixing https://github.com/feathr-ai/feathr/issues/687
         "deltalake<=0.5.8",
         "graphlib_backport<=1.0.3",
-        "protobuf==3.*",
+        "protobuf==3.19.4",
```

Comment:
Can we have something like >= or <= rather than fixing it to a certain version?

Reply:
We cannot make sure if all versions above or below a certain version can work for now.

Reply:
I don't think we have hard requirements for the protobuf 3 version (3.19.4), since we are just using it generically. Having this the strict version will be hard to run in multiple notebook services like synapse/databricks.